CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+To make edits to this site, just click the edit icon on a page on the live site to launch the GitHub code editor.
+If you are a member of the lab you can commit straight to `master` and for 95% of changes that is what you should do;
+do not bother submitting pull requests for this repo; we can always revert if we need to.
+
+After committing to `master`, the site will rebuild and redeploy to https://neuro.polymtl.ca/ within a minute or so. 
+You can watch its progress on the [Actions Tab](https://github.com/neuropoly/neuro.polymtl.ca/actions/workflows/publish.yml)!
```

Comment:
We should make sure this doesn't mess up the sphinx build.

This file isn't in any `toctree` so I think sphinx should just ignore it, but we better double-check.